pre-edited-cs/data/Stage/ScriptSource/Almond.txt

```diff
@@ -90,7 +84,8 @@ ERROR<NOD<END
 You can't get it loose...<NOD<END
 #0243
 <EVE0080
-
+#0244
+<EVE0243
```

Comment:
what is the point of this

Reply:
This was supposed to be part of a fix for getting locked inside the Core room when grabbing the Tow Rope location after the drowning cutscene. But you're right, that event isn't necessary (I can just jump directly to event 0243, I think)

Reply:
that's correct